### PR TITLE
[Improvement-11293][dolphinscheduler-api] remove some variables that redundant in subclasses of AbstractAuthenticator

### LINKED ISSUE #11293: [Improvement][dolphinscheduler] Some variables in subclasses of AbstractAuthenticator are redundant
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

image

image

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuchanghai**
## Purpose of the pull request

remove some variables that redundant in subclasses of AbstractAuthenticator

## Brief change log

remove some variables that redundant in subclasses of AbstractAuthenticator
## Verify this pull request

This pull request is code cleanup without any test coverage.

 this close #11293

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/security/impl/AbstractAuthenticator.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public abstract class AbstractAuthenticator implements Authenticator {
     private static final Logger logger = LoggerFactory.getLogger(AbstractAuthenticator.class);
 
     @Autowired
-    private UsersService userService;
+    protected UsersService userService;
```
Comment (EricGao888) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/security/impl/AbstractAuthenticator.java:44]: What about renaming the Class `UsersService` with `UserService`? Looks kinda wired.

**Review by EricGao888 — APPROVED** [review]

**Comment by ruanwenjun** [comment]
There are so many commits, please rebase from upstream dev when you submit PR next time.

**Review by davidzollo — APPROVED** [review]
+1

**Comment by ruanwenjun** [comment]
@davidzollo Please optimize the commits title and log next time merging a PR
image

**Comment by fuchanghai** [comment]
@ruanwenjun Thanks for reminding

**Pull request merged**